app/helpers/application_helper.rb

```diff
@@ -1,2 +1,21 @@
 module ApplicationHelper
+  def user_or_guest
+    session[:user_id] ? User.find(session[:user_id]).username : "Guest"
```

Comment:
One question, if you use `User.find` and it doesn't find a user, are you allowed to call `.username` on the returned object?